zhaquirks/tuya/ts0121_plug.py

```diff
@@ -31,6 +31,14 @@ class OnOffRestorePowerCluster(CustomCluster, OnOff):
     attributes.update({0x8002: ("power_on_state", PowerOnState)})
 
 
+class TuyaMeteringCluster(CustomCluster, Metering):
+    """TuyaMeteringCluster divides the kWh for tuya."""
+    
+    MULTIPLIER = 0x0301
+    DIVISOR = 0x0302
+    _CONSTANT_ATTRIBUTES = {MULTIPLIER: 0.01, DIVISOR: 100}
```

Comment:
Multiplier and divisor should both be of integer type and since both multiplier and divisor apply you probably want to set multiplier to 1, not to 0.01, for the desired result.

Reply:
Hm, you're right. In my first try it did not work with multiplier 1, but now it's working. Maybe still in cache or whatever... Thanks 😃. Its now the same as in #1057. Should it be moved to init.py?